x-pack/plugin/inference/src/yamlRestTest/resources/rest-api-spec/test/inference/30_semantic_text_query.yml

```
@@ -0,0 +1,348 @@
+setup:
+  - skip:
+      version: " - 8.12.99"
```

```suggestion
      version: " - 8.14.99"
```

... although we may do that in one pass on the feature branch